microservices/workers/src/Worker/Cgrates.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Worker;
+
+use GearmanJob;
+use Ivoz\Core\Infrastructure\Domain\Service\Cgrates\ReloadService;
+use Ivoz\Core\Infrastructure\Domain\Service\Gearman\Jobs\Cgrates as CgratesJob;
+use Mmoreram\GearmanBundle\Driver\Gearman;
+use Symfony\Bridge\Monolog\Logger;
```

Comment:
Psr\Log\LoggerInterface would be a better option (plus it's autowireable)

Reply:
Much more autowireable, yes :+1: